transport-classes-io_uring/src/main/java/io/netty5/channel/uring/IOUringHandler.java

```diff
@@ -212,6 +213,45 @@ public void destroy() {
         }
     }
 
+    // We need to prevent the race condition where a wakeup event is submitted to a file descriptor that has
+    // already been freed (and potentially reallocated by the OS). Because submitted events is gated on the
+    // `eventfdAsyncNotify` flag we can close the gate but may need to read any outstanding events that have
+    // (or will) be written.
```

Comment:
I don't know of a good way to test this. Any suggestions appreciated.

Reply:
Consider that Fd reuse can make this happen as well (which is another bug I have fixed in the incubator) but I have no idea how to trigger it if not by keep on closing fds which forcibly cause a close/write delay. We had a vertx test able to reproduce it but I failed to understand how it was possible with a purely Netty reproducer